pkg/controlplane/controller/clusterauthenticationtrust/cluster_authentication_trust_controller.go

```diff
@@ -259,6 +262,10 @@ func getConfigMapDataFor(authenticationInfo ClusterAuthenticationInfo) (map[stri
 		if err != nil {
 			return nil, err
 		}
+		data["requestheader-uid-headers"], err = jsonSerializeStringSlice(authenticationInfo.RequestHeaderUIDHeaders.Value())
```

Comment:
this may be causing dueling / ~hotloop configmap writes when 1.31 and 1.32 API servers interact, see https://github.com/kubernetes/kubernetes/issues/128986#issuecomment-2516536495

Reply:
😢 😭